### DISCUSSION TITLE
RFC - Remove redundant `amplication-` prefix in our monorepo package folders

### DISCUSSION THREAD

**Discussion by arielweinberger** [amplication/amplication#3161] [Ideas]
# Motive
Currently, all of our monorepo package folder names start with `amplication-*`. My suggestion is to remove the `amplication-` prefix as it is redundant. It will be cleaner.

I'm writing this suggestion after investigating other common open-source monorepos, and I can clearly see that they do not include their org name in package folders.

Examples:
- [nestjs/nest](https://github.com/nestjs/nest/tree/master/packages)
- [nrwl/nx](https://github.com/nrwl/nx/tree/master/packages)
- [angular/angular](https://github.com/angular/angular/tree/main/packages)
- [react/react](https://github.com/facebook/react/tree/main/packages) - some exceptions since it also contains non-react packages
- [vue/vue](https://github.com/vuejs/vue/tree/main/packages)

# Suggestion
Rename our `package/*` subfolders to remove the `amplication-` prefix.
The package name in `package.json` is still under the `@amplication` organization.

Currently:
```
/ee
  /packages
    /amplication-git-pull-service
    /amplication-git-push-webhook-service
/packages
  /amplication-cli
  /amplication-client
  /amplication-data
  /amplication-data-service-generator
  /amplication-design-system
  /amplication-git-pull-request-service
  /amplication-git-service
  /amplication-prisma-db
  /amplication-scheduler
  /amplication-server
  /amplication-storage-gateway
```

Proposed:
```
/ee
  /packages
    /git-pull-service
    /git-push-webhook-service
/packages
  /cli
  /client
  /data
  /data-service-generator
  /design-system
  /git-pull-request-service
  /git-service
  /prisma-db
  /scheduler
  /server
  /storage-gateway
```
```

**Comment by arielweinberger** [comment]
@amplication/amplication 
Your feedback is needed :)

**Comment by tupe12334** [comment]
I can't agree with that more! I think its a small change and needed for a better professional repository

**Comment by yuval-hazaz** [comment]
looks good to me
